docs/features/transformers.md

```diff
@@ -56,16 +56,24 @@ Pulls a field out of a complex value and replaces the entire value with the extr
       field: id
 ```
 
-## Split Value
-Split an array into values.
+## Split array
+Split an array into multiple records.
```

Comment:
Could you please explain in more detail in documentation what is expected behavior of this transformer?
"jmesPath" contains a path to the array. But I didn't get what is the purpose of the "to" configuration property.

Reply:
I'll put more info on the general behavior

Reply:
Much better now, thanks.